great_expectations/compatibility/databricks.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
```

Comment:
Can we remove this now that we're not testing using this method?

Reply:
Looks like the answer is yes. I was actually planning to remove this entire module now that we don't reference it anywhere.